### ISSUE TITLE
Feature request: filter by review state

### ISSUE THREAD

**Issue by keradus** [chinthakagodawita/autoupdate#195]
There is a possibility to auto-update PRs that are tagged with concrete labels. Another one is for non-draft PRs.

Can the new filtering be added, that auto-update would trigger only for PRs that got already reviewed?
I would love to not waste CPU power of CI to run on each auto-update merge for PR that is not yet reviewed

**Comment by chinthakagodawita** [comment]
Hey @keradus - I don't think this would save CI minutes, the 'PR evaluation' step barely takes a second to run per-PR.

It's also difficult to tell what 'reviewed' means as it's highly dependent on your review workflow, some workflows may require a certain number of reviews, others wont.

Perhaps an action that auto-labels your PRs would help? e.g. https://github.com/marketplace/actions/label-approved-pull-requests

In other words:
- A certain number of reviews happen
- That action automatically adds the `autoupdate` label to your PR
- autoupdate will evaluate this PR on its next run

**Comment by keradus** [comment]
- the "review" state is managed by GitHub, right? whatever rule one applies for the PR, GitHub knows it and expose the state whether PR is reviewed or not.
- I think we got into misunderstanding of CI CPU waste - you think about CPU consumed by autoupdate GitHub action, right? I'm thinking about CPU consumed for work triggered by new commit created by autoupdate action.

would you please reconsider the request?

**Comment by chinthakagodawita** [comment]
Hey @keradus sorry for the late reply on this!

> the "review" state is managed by GitHub, right? whatever rule one applies for the PR, GitHub knows it and expose the state whether PR is reviewed or not.
This isn't something that Github exposes through the API unfortunately. You can see the full set of pull request data we have access to here: https://docs.github.com/en/rest/reference/pulls#get-a-pull-request

> I'm thinking about CPU consumed for work triggered by new commit created by autoupdate action.
This should be insignificant compared to the amount resource used when autoupdate loads and compares PRs. autoupdate doesn't do the merge manually, it calls out to Github's API to do this.
